tools/libSwiftScan/libSwiftScan.cpp

```diff
@@ -57,11 +71,33 @@ class SwiftScanCAS {
   std::shared_ptr<llvm::cas::ObjectStore> CAS;
   std::shared_ptr<llvm::cas::ActionCache> Cache;
 };
+
+struct SwiftCachedCompilation {
```

Comment:
Would it be fair to call this a `SwiftCachedCompilationHandle`? Otherwise my first intuition is that this contains some kind of compilation result **state**.